conftest.py

```diff
@@ -4,18 +4,27 @@
 from account.models.client import Client
 from chatmessage.models import Chatmessage, SenderType
 from datetime import datetime
+from django.utils import timezone
 import pytest
 
 
 USERNAME = "testusername"
+USERNAME2 = "testusername2"
+USERNAME3 = "testusername3"
 PASSWORD = "testpassword"
+PASSWORD2 = "testpassword2"
+PASSWORD3 = "testpassword3"
 FIRST_NAME = 'Bob'
 LAST_NAME = 'Builder'
 EMAIL = "test@test.com"
-LAST_LOGIN = datetime.now()
+EMAIL2 = "test@test2.com"
+EMAIL3 = "test@test3.com"
+LAST_LOGIN = timezone.now()
 
 PROFILE_TYPE = 'C'
 PHONE_NUMBER = '123456789'
```

Comment:
Please use Array for all the sequences
In the tests uses PHONE_NUM_ARR[1] instead of using PHONE_NUMBER2 
and the same goes for all varaible arrays

Reply:
I've changed it to a different way:
4 global dictionaries for the fields in each entity and the value of each key is a list
so instead of PHONE_NUM_ARR[1] we will use PROFILE_INFORMATION.get('phone_number')[1]
because 'phone_number' its a a field in the Profile entity
yes its longer, but I think that its more correct way to use global values and keep the code in OOP :)
@Yarboa